certora/conf/GuardianStorage.conf

```diff
@@ -0,0 +1,26 @@
+{
+    "files": [
+        "contracts/modules/social_recovery/storage/GuardianStorage.sol",
+        "contracts/modules/social_recovery/SocialRecoveryModule.sol",
+        "certora/harnesses/SafeHarness.sol"
+    ],
+    "msg": "GuardianManager: General Ruleset",
+    "rule_sanity": "basic",
+    "loop_iter": "3",
+    "optimistic_loop": true,
+    "hashing_length_bound": "352",
+    "optimistic_hashing": true,
+    "solc": "solc-0.8.20",
+    "coverage_info": "basic",
+    "prover_args": [
+        "-smt_groundQuantifiers true -depth 0"
+    ],
+    "verify": "GuardianStorage:certora/specs/GuardianStorage.spec",
+    "parametric_contracts" : [
+        "GuardianStorage"
```

Comment:
@remedcu introduced a `GuardianStorageHarness` - should we use that here as well?

Reply:
`GuardianStorageHarness` has additional `countGuardians(...)` function which is not required here.

Reply:
OK - lets leave it for #25 then, and keep it as-is here.